include/tracewrap.h

```diff
@@ -23,12 +17,12 @@ void qemu_trace_finish(uint32_t exit_code);
 OperandInfo * load_store_reg(target_ulong reg, target_ulong val, int ls);
```

Comment:
These two functions repeats in all backends with mostly the same contents, it is tempting to move them in `tracewrap.c`